pkg/cmd/controller/start.go

```diff
@@ -0,0 +1,88 @@
+package controller
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	clientset "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned"
+	informers "github.com/xmudrii/etcdproxy-controller/pkg/client/informers/externalversions"
+	"github.com/xmudrii/etcdproxy-controller/pkg/controller/etcdproxy"
+	"github.com/xmudrii/etcdproxy-controller/pkg/options"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+)
+
+func NewCommandEtcdProxyControllerStart(stopCh <-chan struct{}) *cobra.Command {
+	o := options.NewEtcdProxyControllerOptions()
+
+	cmd := &cobra.Command{
+		Short: "Start EtcdProxyController",
+		Long:  "Start EtcdProxyController",
+		RunE: func(c *cobra.Command, args []string) error {
```

Comment:
I discovered that this prints a pretty useless usage message.  I refactored my command to simply `glog.Fatal` on errors

Reply:
Same for me. I think this should be fixed now.